install.sh

```diff
@@ -56,7 +56,7 @@ download_safeu_cli() {
 }
 
 install_scope() {
-    printf "${YELLOW}Install safeu command tool globally (require sudo permission later) ? [Y/N, Defualt: Y]: "
+    printf "${YELLOW}Install safeu command tool globally (require sudo permission later) ? [Y/N, Default: Y]: "
```

Comment:
What’s the difference?

Reply:
A typo at the end of this prompt, def**au**lt vs def**ua**lt.

Though, there is a common way for expressing such choice, for example `[Y/n]`, `[no/YES/skip]`, where the UPPERCASE is default choice.

Reply:
Okay, if I use the `[Y/N]`, how to represent the default value? Or using a infinite input loop no stop until read `Y` or `N`?